app/templates/eventdetailsformtab.html

```diff
@@ -0,0 +1,48 @@
+<div class="form_row">
```

Comment:
Change the name to event_details_form_tab.html

Reply:
Will fix :)